app/views/works/index.html.erb

```diff
@@ -0,0 +1,83 @@
+<h2> List of Works </h2>
+
+<section class="media-table">
+    <h4> Books </h4>
+
+    <table class='table'>
+        <thead>
+            <tr>
+                <th width='80'>Votes</th>
+                <th>Title</th>
+                <th>Created By</th>
+                <th width='100'>Published</th>
+                <th width='100'>Upvote</th>
+            </tr>
+        </thead>
+        <tbody>
+            <% @books.each do |book| %>
+                <tr>
+                    <td><%= book.votes.count %> votes </td> 
```

Comment:
You repeat this work 3 times. Seems like a good opportunity to use a view partial!